ecommerce/api.py

```diff
@@ -96,8 +97,10 @@ def get_purchasable_course_run(course_key, user):
             order__user=user,
             course_key=course_run.edx_course_key,
```

Comment:
so for a course user can attempt 1 time exam in one payment, the for each new exam authorization he needs to  purchase the run